.pipelines/cg-pipeline.yaml

```diff
@@ -43,4 +43,105 @@ stages:
               fi
             name: "EnvironmentalVariables"
             displayName: "Set environmental variables"
-            condition: always()
\ No newline at end of file
+            condition: always()
+
+      - job: retinaagentimages
+        displayName: Build Retina Agent Images
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        strategy:
+          matrix:
+            agent-linux-arm:
+              platform: "linux"
+              arch: "arm64"
+              target: "agent"
+
+            agent-linux-amd64:
+              platform: "linux"
+              arch: "amd64"
+              target: "agent"
+
+            init-linux-arm:
+              platform: "linux"
+              arch: "arm64"
+              target: "init"
+
+            init-linux-amd64:
+              platform: "linux"
+              arch: "amd64"
+              target: "init"
+            
+            operator-linux-amd64:
```

Comment:
in the github actions there's a separate job, maybe follow that if agent/operator need separate matrices for makefile commands, if that's easier

https://github.com/microsoft/retina/blob/main/.github/workflows/images.yaml#L109